dist/index.html

```diff
@@ -10,6 +10,7 @@
 
 <body>
   <h1 id="headerText">Activity Tracker</h1>
+  <h1 class="current-date" id="currentDate"></h1>
```

Comment:
We should change this to an h2 or another tag so we don't have multiple h1 headers. But that will be an easy fix!

This is also a great spot for us to add the inputs when we get our POSTs up and running.